4.analyze-components/scripts/nbconverted/5.visualize-svcca.r

```diff
@@ -31,52 +34,117 @@ svcca_df$algorithm_2 <- factor(
 
 options(repr.plot.width = 15, repr.plot.height = 9)
 
-for (dataset in c("TARGET", "TCGA", "GTEX")) {
+for(dataset in c("TARGET", "TCGA", "GTEX")) {
```

Comment:
looks like you have 3 for loops here:) Not sure how long this takes to run, curious if you would want to always run all three datasets everytime?

Reply:
yeah, I am just plotting and I will want to do it for each dataset each time